scripts/rmc_flags.py

```diff
@@ -127,6 +127,8 @@ def add_developer_flags(make_group, add_flag, config):
              help="Pass through directly to CBMC; must be the last flag")
     add_flag(group, "--mangler", default="v0", choices=["v0", "legacy"],
              help="Change what mangler is used by the Rust compiler")
+    add_flag(group, "--abs-type", default="std", choices=["std", "rmc", "c-ffi", "no-back"],
+             help="Choose abstraction for modules of standard library if available")
```

Comment:
Right now, we're only supporting a single choice, even if they have multiple data-structures that need to be abstracted.